File: back/back/DtoModels/MessageDTO.cs
using System;

namespace back.DtoModels
{
    public class MessageDTO
    {
        public string clientuniqueid { get; set; }
        public string senderUsername { get; set; }
        public string receiverId { get; set; }
        public string receiverUsername { get; set; }
        public string type { get; set; }
        public string message { get; set; }
        public DateTime date { get; set; }
    }
}

File: front/src/app/models/MessageDTO.ts
export class MessageDTO{
    clientuniqueid:string;
    senderUsername:string;
    receiverId:string;
    receiverUsername:string;
    type:string;  
    message: string;  
    date:Date;  

    // constructor(clientuniqueid, type, message, ReceiverUsername, date) {
    //   this.clientuniqueid = clientuniqueid;
    //   this.type = type;
    //   this.message = message;
    //   this.date = date;
    // }
  
}